docs/User Manual.rst

```diff
@@ -185,21 +185,41 @@ It is a type of volumetric source. For the moment it is constrained to neutrons.
 The properties of a material source are:
 
 * mat: the name of the material from which to sample (must be defined in materials).
-* data (*optional*, default = continuous energy): data type for source particles. Can be ``ce`` 
+* data (*optional*, default = continuous energy): data type for source particles. Can be ``ce``
   or ``mg``.
-* E (*optional*, default = 1E-6): energy of the particles emitted, for continuous energy 
+* E (*optional*, default = 1E-6): energy of the particles emitted, for continuous energy
   calculations. [MeV]
-* G (*optional*, default = 1): energy group of the particles emitted, for multi-group 
+* G (*optional*, default = 1): energy group of the particles emitted, for multi-group
   calculations.
-* boundingBox (*optional*, default is the geometry bounding box): 
-  (x_min y_min z_min x_max y_max z_max) vector describing a bounding box to improve sampling 
+* boundingBox (*optional*, default is the geometry bounding box):
+  (x_min y_min z_min x_max y_max z_max) vector describing a bounding box to improve sampling
   efficiency or to localise material sampling to a particular region.
 
 Hence, an input would look like: ::
 
-      source { type materialSource; mat myMat; data ce; E 2.0; 
+      source { type materialSource; mat myMat; data ce; E 2.0;
       boundingBox (-5.0 -3.0 2.0 5.0 4.0 3.0); }
 
+fissionSource
+#############
+
+A source intended to be used in eigenvalue calculations. It is a type of volumetric source, which
+uniformly distributes fission sites in the geometry. The energy spectrum of the fission neutrons
+is based on a fixed incident energy provided by the user. The properties of a fission source are:
```

Comment:
Maybe the first sentence is a bit misleading, it sounds like one should explicitly define it when doing an eigenvalue calculation. We could clarify a bit? This is just a proposal:

```suggestion
A source intended to initialise eigenvalue calculations. If it is not defined in the input file, it is 
used with the default settings. It is a type of volumetric source, which uniformly distributes fission
sites in the geometry. The energy spectrum of the fission neutrons is based on a fixed incident 
energy provided by the user. The properties of a fission source are:
```

Reply:
Sure! Commited (and removed trailing whitespace)